### PR TITLE
[yt/python] Include prefix "YT_" for names of SDK environment variables

### PR DESCRIPTION
**Pull request by koct9i**
Make grep-friendly mentions of all known environment variables.

Most lines of patch are generated by this:
sed -iE 's/    "\([A-Z0-9_]\+\)"/    "YT_\1"/' yt/python/yt/wrapper/default_config.py

### CODE REVIEW

File: yt/python/yt/wrapper/default_config.py [hunk 1]
```diff
@@ -720,87 +720,87 @@ def get_default_config():
 
 
 FORCED_SHORTCUTS = {
-    "BASE_LAYER" : "operation_base_layer",
+    "YT_YT_BASE_LAYER" : "operation_base_layer",
```
Comment (gritukan) [inline yt/python/yt/wrapper/default_config.py:723]: YT_YT?

File: yt/python/yt/wrapper/default_config.py [hunk 2]
```diff
@@ -865,7 +865,6 @@ def _get(d, key):
         if not key.startswith(prefix):
```
Comment (gritukan) [inline yt/python/yt/wrapper/default_config.py:865]: Do we need this code after the change?

**Comment by Kontakter** [comment]
Please, read the code carefully, there are some other places that should be accurately fixed after this change: at least this one https://github.com/ytsaurus/ytsaurus/blob/main/yt/python/yt/wrapper/default_config.py#L988

**Review by Kontakter — CHANGES_REQUESTED** [review]
See my and @gritukan  comments

File: yt/python/yt/wrapper/default_config.py (hunk 2 above)
Comment (koct9i) [reply yt/python/yt/wrapper/default_config.py:865]: Looks like good sanity-check and optimization - all must start with "YT_"

**Comment by koct9i** [comment]
done

**Comment by koct9i** [comment]
found one more place with obscure environment references

**Review by Kontakter — APPROVED** [review]
To run test in internal CI

**Pull request closed without merging**